java/src/hu/holyoil/crewmate/Settler.java

```diff
@@ -14,10 +14,10 @@
  * Leszármazottja az AbstractCrewmate-nek (robottal közös tulajdonságai miatt).
  * Implementálja az IStorageCapable-t mert képes tárolásra és gyártásra.
  */
-public class Settler extends AbstractCrewmate implements IStorageCapable {
+public class Settler extends AbstractCrewmate implements IStorageCapable, IMiner {
     /**
      * Létrehoz egy telepest
-     * <p>Kívülről nem elérhető: nem lehet kezdő Aszteroida nélkül példányosítani.
+     * <p>Kívülről nem elérhető: nem lehet kezdő storage nélkül példányosítani.
```

Comment:
Why did this get changed? I believe the previous version was correct.

Reply:
true (technically both are correct.) I'll change it back